build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -50,8 +50,14 @@ Future<Iterable<Expression>> _findBuilderApplications(String configKey) async {
     if (buildConfigOverrides.containsKey(package.name)) {
       return buildConfigOverrides[package.name];
     }
-    return BuildConfig.fromBuildConfigDir(
-        package.name, package.dependencies.map((n) => n.name), package.path);
+    try {
+      return await BuildConfig.fromBuildConfigDir(
+          package.name, package.dependencies.map((n) => n.name), package.path);
+    } catch (e) {
```

Comment:
could this really be anything? catch alls make me nervous

Reply:
catching `ArgumentError` now.